### PR TITLE
Add description field to the EIP header

### PR DESCRIPTION
**Pull request by axic**
Implementation of https://github.com/ethereum/EIPs/pull/3564#issuecomment-837367526.

The header:
Screenshot

Twitter before:
Screenshot

Twitter after:
Screenshot

**Could also render the `description` prominently after the header.**

### CODE REVIEW

File: EIPS/eip-1.md
Comment (MicahZoltu) [inline EIPS/eip-1.md:122]: Consider adding this text (from the template):
> Provide a simplified and layman-accessible explanation of the EIP. Imagine an email subject line, GitHub PR title, or forum post title.

File: eip-template.md
Comment (MicahZoltu) [inline eip-template.md:4]: Recommend putting this text in here:
> Provide a simplified and layman-accessible explanation of the EIP. Imagine an email subject line, GitHub PR title, or forum post title.

People really struggle with "simple" I have found, and giving them extra guidance is valuable I think.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Recommend removing the Simple Summary from the template and EIP-1, as this would be the new place to put that content.

I generally approve, but would like to see the Simple Summary removed along with the introduction of a description metadata field.

**Comment by axic** [comment]
> Recommend removing the Simple Summary from the template and EIP-1, as this would be the new place to put that content.
>
> I generally approve, but would like to see the Simple Summary removed along with the introduction of a description metadata field.

Absolutely, that would be the goal, but decided to keep the first step simple until there's a general agreement on this.

> Could also render the description prominently after the header.

My main question right now is how to render it on the page:
1) only part of the header (as in the PR now)
2) before the header in large text
3) after the header in large text
4) add a new section (like simple summary), but call it Description
5) both 1) and one of 2) to 4)

File: eip-template.md
Comment (axic) [reply eip-template.md:4]: Yes, it should be much better phrased. However "imagine a subject line" is confusing, as I suppose the `title` should be that.

Comment (MicahZoltu) [reply eip-template.md:4]: Title should be shorter than a subject line.  Title is more like movie title, one or two words.  Subject line is more like a description, about a sentence.

**Comment by MicahZoltu** [comment]
I care less about answering the "how it is rendered" question right now, as that is something we can pretty easily/freely iterate on after this change I think.  I'm fine with starting out with something simple and then improving it later.

**Comment by MicahZoltu** [comment]
> Absolutely, that would be the goal, but decided to keep the first step simple until there's a general agreement on this.

I would like to see that happen in this PR, not a future PR.  I dislike the idea of having a period of time where we have both a `description` and a `## Simple Summary`

**Comment by axic** [comment]
> > Absolutely, that would be the goal, but decided to keep the first step simple until there's a general agreement on this.
>
> I would like to see that happen in this PR, not a future PR. I dislike the idea of having a period of time where we have both a description and a ## Simple Summary

I planned to do it in this PR, but did not wanted to bring extra work to myself before this field is agreed on.

**Comment by axic** [comment]
Some variations:

1)
image

2)
Screenshot

3)
Screenshot

**Comment by MicahZoltu** [comment]
I like 2.

**Comment by lightclient** [comment]
I like both 1 and 2. In terms of the actual text in the templates, I don't mind too much. I'm on board with the change in general though (it will be great to not repeat the EIP number a dozen times in the social thumbnail!)

**Review by lightclient — APPROVED** [review]

File: EIPS/eip-1.md
Comment (axic) [reply EIPS/eip-1.md]: Done.

File: eip-template.md
Comment (axic) [reply eip-template.md]: Done.

**Comment by axic** [comment]
@MicahZoltu should be ready to merge, but let me know if you want to remove the commit for 2718.

File: EIPS/eip-2718.md
Comment (MicahZoltu) [inline EIPS/eip-2718.md:4]: Simple Summary should be removed with the inclusion of description. I think.

File: eip-template.md
Comment (MicahZoltu) [inline eip-template.md:4]: I think it would be valuable to include this text or similar, but not a hard requirement if others disagree:
> Imagine an email subject line, GitHub PR title, or forum post title.

File: EIPS/eip-2718.md
Comment (axic) [reply EIPS/eip-2718.md:4]: Right. My question was do you want to touch this file in this PR or not?

File: eip-template.md
Comment (axic) [reply eip-template.md:4]: I don't want to argue too much about it, but what is the title then?

File: EIPS/eip-2718.md
Comment (MicahZoltu) [reply EIPS/eip-2718.md:4]: I'm fine with 2718 getting updated as a "first example" in this PR.

Comment (axic) [reply EIPS/eip-2718.md:4]: Ok, also removed the section.

File: eip-template.md
Comment (MicahZoltu) [reply eip-template.md:4]: Title is a few words, not a complete sentence.
> King Micah

Description is one full (short) sentence.
> The benevolent dictator of the EIPs repository.

Abstract is multi-sentence (short paragraph) technical summary.
> Micah is the unelected, self assigned dictator of the EIPs repository.  They rule by way of blocking an PRs they don't like and merging things they like without waiting for agreement from anyone else.  Micah's power can be taken away by someone else volunteering to take on the responsibilities of shepherding the EIP's repository."

Another way to look at it is that a Title is what you would use in a sentence when referring to the EIP.  "We should implement the `EVM Object Format` in Shanghai."  The description is what you would use to jog someone's memory as to what the EIP is.  "What is the `EVM Object Format`?" "It is the EIP that `Establishes contract versioning by way of a leading bytecode sequence.`" "Ah, right, I remember that one."

Comment (axic) [reply eip-template.md:4]: I think actually your description is much better suited as opposed to the examples you gave :)

```
title: <Title is a few words, not a complete sentence.>
description: <Description is one full (short) sentence.>
abstract: <Abstract is multi-sentence (short paragraph) technical summary.>
```

Comment (MicahZoltu) [reply eip-template.md:4]: Works for me.  :)

**Review by MicahZoltu — APPROVED** [review]

**Comment by axic** [comment]
@MicahZoltu this still needs a manual merge from you 😅

**Comment by axic** [comment]
Blocked by https://github.com/lightclient/eipv/pull/28 😭 

All eyes are on you @lightclient 🙏

File: EIPS/eip-1.md
Comment (lightclient) [inline EIPS/eip-1.md:100]:
```suggestion
- Abstract - Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
```

File: eip-template.md
Comment (lightclient) [inline eip-template.md:22]:
```suggestion
Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
```

File: EIPS/eip-1.md
Comment (axic) [reply EIPS/eip-1.md]: Added.

File: eip-template.md
Comment (axic) [reply eip-template.md]: Added.

File: EIPS/eip-1.md
Comment (SUZETTEWIL) [inline EIPS/eip-1.md]: It was ERC20 tokens send to my luno ethereum wallet. Was stuck on etherscan and they released the tokens for public.

File: EIPS/eip-1.md
Comment (SUZETTEWIL) [inline EIPS/eip-1.md:124]: MISAH ZOITU

File: EIPS/eip-1.md
Comment (SUZETTEWIL) [inline EIPS/eip-1.md]: {
qrcode_user-images githubusercontent com

File: _includes/head.html
Comment (SUZETTEWIL) [inline _includes/head.html:6]: _https://github.com/ethereum/EIPs/pull/3706/commits/4f2a26e4d2f12c0676c664053ceb1c97aeaa26a8#diff-e241bda4e3c3c6dc1c0b00185b61f6ce19b5eb16e294dd955ca9fa6d01befb0e

File: _includes/head.html
Comment (SUZETTEWIL) [inline _includes/head.html:5]: https://github.com/ethereum/EIPs/pull/3706/commits/4f2a26e4d2f12c0676c664053ceb1c97aeaa26a8#diff-e241bda4e3c3c6dc1c0b00185b61f6ce19b5eb16e294dd955ca9fa6d01befb0e

**Pull request merged**
